### PR TITLE
Home Page content inclusion page

### LINKED ISSUE #9666: Add documentation page for Home Page content requests
I feel like it would be good for us to add some documentation on how and what we need to add new cards and banners to the Home Page.  I keep getting this question...

Maybe put it under "Miscellaneous" under a "Home Page Content" page?
image

Here's a start - 

If you have a tutorial, project, lesson or full course that you would like to share with others, we're happy to list it as a resource on the Home Page.  Please read the information below to learn more about the request process.

**Criteria**
- Your content should be open source and free to use
- The instructions should be well written and easy to follow
- It should not include any personally identifiable information
- Content will be reviewed and approved by the MakeCode Team

**What you need to submit**
- For a tutorial, a link to a complete tutorial (see https://makecode.com/writing-docs/tutorials for more details)
- For a lesson or course, a link to a public website with the materials
- An image of <insert dimensions and/or resolution here>
- A name and short description - see below as an example
image

**Process**
- File a GitHub issue requesting your content to be listed
- Include all the relevant information listed above
- For micro:bit, please follow the instructions for submitting a tutorial to the Micro:bit Foundation for approval [here](https://support.microbit.org/support/solutions/articles/19000054952-makecode-extension-and-tutorial-approval)

For requesting a featured space in the Home Page banner carousel, we require a link, a short description and an image of <insert dimensions and/or resolution here>.  Please note that banner space is very limited, and therefore highly selective.

### PR DESCRIPTION
**Pull request by ganicke**
This is a draft of a base page for content additions of user content to a target home page. This is a base page and is extended in the target doc space for target specific images and approval info.

Closes #9666

### CODE REVIEW

File: common-docs/homepage-content.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Much of content available on the home page is developed or selected by the MakeCode Team. Content from other contributors is welcomed though and can be featured in the home page galleries.
+
+## #target-homescreen
+
+## Content requirements
+
+* Your content should be open source and free to use without any restrictions other than author attribution (you may specify an open source license the provides for such usage if you wish).
+* The instructions should be well written and easy to follow.
+* It should not include any personally identifiable.
```
Comment (ganicke) [inline common-docs/homepage-content.md:13]: No attribution info?

File: common-docs/homepage-content.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Adding content to the home page
+
+The Editor home page contains galleries of card links to projects, tutorials, lessons, videos, and other content sources. Many content items featured on the home page are hosted at the editor's website while some other items are from external sources.
+
+Much of content available on the home page is developed or selected by the MakeCode Team. Content from other contributors is welcomed though and can be featured in the home page galleries.
+
+## #target-homescreen
```
Comment (ganicke) [inline common-docs/homepage-content.md:7]: Example card image/gallery placement for user content link.

File: common-docs/homepage-content.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+### Description
+
+Learn how to create moving sprites that randomly explode!
+
+### Thumbnail
+
+![image](https://github.com/microsoft/pxt/assets/27789908/f5c21294-1145-4010-8de0-560b8afbfeec)
+```
+
+## #target-approval
```
Comment (ganicke) [inline common-docs/homepage-content.md:46]: Approval info, if any, override for target page.

File: common-docs/homepage-content.md (hunk 1 above)
Comment (Jaqster) [reply common-docs/homepage-content.md:13]: Sorry, what's the question?

Comment (Jaqster) [inline common-docs/homepage-content.md:13]: "personally identifiable information"

**Review by Jaqster — APPROVED** [review]
Looks great!  Thanks Galen!

**Pull request merged**